core/src/main/python/wlsdeploy/util/cla_utils.py

```diff
@@ -70,6 +70,7 @@ class CommandLineArgUtil(object):
     ATTRIBUTES_ONLY_SWITCH     = '-attributes_only'
     FOLDERS_ONLY_SWITCH        = '-folders_only'
     RECURSIVE_SWITCH           = '-recursive'
+    UPDATE_RCU_SCHEMA_PASS_SWITCH = '-updateRCUSchemaPassword'
```

Comment:
Why is this new switch necessary?  Why isn't the presence of the new domainInfo attribute sufficient?

Reply:
This is in WDT which has no idea whether the information is new or old.